{"uid":"2582cae8e18d6ebb","name":"FAILED 기능 동작 하는지 확인","fullName":"src.tests.ui_tests.test_fail_screenshot#test_fail_screenshot","historyId":"295b0300b44e64aac7e9de8f2dd396ad","time":{"start":1746140634332,"stop":1746140634332,"duration":0},"description":"\n일부러 실패하게 만드는 테스트\n- 올바르지 않은 요소 선택\n- 검색창이 없는 요소를 클릭 시도하여 실패 유발\n","descriptionHtml":"<p>일부러 실패하게 만드는 테스트</p>\n<ul>\n<li>올바르지 않은 요소 선택</li>\n<li>검색창이 없는 요소를 클릭 시도하여 실패 유발</li>\n</ul>\n","status":"broken","statusMessage":"IndexError: tuple index out of range","statusTrace":"self = <webdriver_manager.core.file_manager.FileManager object at 0x7f82d1fbb8c0>\narchive_file = <webdriver_manager.core.archive.Archive object at 0x7f82d1fba270>\nto_directory = '/root/.wdm/drivers/chromedriver/linux64/136.0.7103.49'\n\n    def __extract_zip(self, archive_file, to_directory):\n        zip_class = (LinuxZipFileWithPermissions if self._os_system_manager.get_os_name() == \"linux\" else zipfile.ZipFile)\n        archive = zip_class(archive_file.file_path)\n        try:\n>           archive.extractall(to_directory)\n\n/usr/local/lib/python3.13/site-packages/webdriver_manager/core/file_manager.py:65: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/usr/local/lib/python3.13/zipfile/__init__.py:1788: in extractall\n    self._extract_member(zipinfo, path, pwd)\n/usr/local/lib/python3.13/zipfile/__init__.py:1850: in _extract_member\n    shutil.copyfileobj(source, target)\n/usr/local/lib/python3.13/shutil.py:203: in copyfileobj\n    while buf := fsrc_read(length):\n/usr/local/lib/python3.13/zipfile/__init__.py:1015: in read\n    data = self._read1(n)\n/usr/local/lib/python3.13/zipfile/__init__.py:1083: in _read1\n    data += self._read2(n - len(data))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <zipfile.ZipExtFile [closed]>, n = 41053\n\n    def _read2(self, n):\n        if self._compress_left <= 0:\n            return b''\n    \n        n = max(n, self.MIN_READ_SIZE)\n        n = min(n, self._compress_left)\n    \n        data = self._fileobj.read(n)\n        self._compress_left -= len(data)\n        if not data:\n>           raise EOFError\nE           EOFError\n\n/usr/local/lib/python3.13/zipfile/__init__.py:1118: EOFError\n\nDuring handling of the above exception, another exception occurred:\n\n    @pytest.fixture(scope=\"module\")\n    def driver():\n        options = Options()\n        options.add_argument(\"--headless\")\n        options.add_argument(\"--disable-blink-features=AutomationControlled\")\n        options.add_argument(\"--no-sandbox\")\n        options.add_argument(\"--disable-dev-shm-usage\")\n        options.add_argument(\"--disable-gpu\")\n        options.add_argument(\"--window-size=1920,1080\")\n        options.add_argument(\"user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36\")\n    \n        # ✅ CI 환경에서는 캐시 삭제로 손상된 드라이버 방지\n        if IS_CI:\n            cache_dir = os.path.expanduser(\"~/.wdm\")\n            if os.path.exists(cache_dir):\n                shutil.rmtree(cache_dir)\n    \n>       service = Service(ChromeDriverManager().install())\n\nsrc/tests/conftest.py:31: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/usr/local/lib/python3.13/site-packages/webdriver_manager/chrome.py:40: in install\n    driver_path = self._get_driver_binary_path(self.driver)\n/usr/local/lib/python3.13/site-packages/webdriver_manager/core/manager.py:41: in _get_driver_binary_path\n    binary_path = self._cache_manager.save_file_to_cache(driver, file)\n/usr/local/lib/python3.13/site-packages/webdriver_manager/core/driver_cache.py:54: in save_file_to_cache\n    files = self.unpack_archive(archive, path)\n/usr/local/lib/python3.13/site-packages/webdriver_manager/core/driver_cache.py:49: in unpack_archive\n    return self._file_manager.unpack_archive(archive, path)\n/usr/local/lib/python3.13/site-packages/webdriver_manager/core/file_manager.py:57: in unpack_archive\n    return self.__extract_zip(archive_file, target_dir)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <webdriver_manager.core.file_manager.FileManager object at 0x7f82d1fbb8c0>\narchive_file = <webdriver_manager.core.archive.Archive object at 0x7f82d1fba270>\nto_directory = '/root/.wdm/drivers/chromedriver/linux64/136.0.7103.49'\n\n    def __extract_zip(self, archive_file, to_directory):\n        zip_class = (LinuxZipFileWithPermissions if self._os_system_manager.get_os_name() == \"linux\" else zipfile.ZipFile)\n        archive = zip_class(archive_file.file_path)\n        try:\n            archive.extractall(to_directory)\n        except Exception as e:\n>           if e.args[0] not in [26, 13] and e.args[1] not in [\n                \"Text file busy\",\n                \"Permission denied\",\n            ]:\nE           IndexError: tuple index out of range\n\n/usr/local/lib/python3.13/site-packages/webdriver_manager/core/file_manager.py:67: IndexError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"afterStages":[],"labels":[{"name":"story","value":"검색창 FAILED 테스트"},{"name":"feature","value":"UI 테스트"},{"name":"parentSuite","value":"src.tests.ui_tests"},{"name":"suite","value":"test_fail_screenshot"},{"name":"host","value":"a31b2ec0eba1"},{"name":"thread","value":"10-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"src.tests.ui_tests.test_fail_screenshot"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"2582cae8e18d6ebb.json","parameterValues":[]}